### PR TITLE
Fixing some warnings

### PR DESCRIPTION
**Pull request by wrexbe**
## About the PR

- Replace SS14Window with DefaultWindow
- Fix some names
- Remove some unused variables
- Unhide, or mark hide as intended,
- Fixed some fixture warnings

### CODE REVIEW

File: Content.Server/Chemistry/ReactionEffects/AreaReactionEffect.cs [hunk 1]
```diff
@@ -47,6 +47,7 @@ public abstract class AreaReactionEffect : ReagentEffect, ISerializationHooks
         /// <summary>
         /// Used to calculate concentration. Reagents get linearly more concentrated as the range goes from
         /// _reagentDilutionStart to zero. When the range is zero the reagents volume gets multiplied by this.
+        /// TODO: This appears to be unused, but maybe should be used, need to investigate
```
Comment (PaulRitter) [inline Content.Server/Chemistry/ReactionEffects/AreaReactionEffect.cs:50]: git blame this & ping the corresponding person tbh

File: Content.Server/GameTicking/GameTicker.Lobby.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ public partial class GameTicker
         private TimeSpan _pauseTime;
 
         [ViewVariables]
-        public bool Paused { get; set; }
+        public new bool Paused { get; set; }
```
Comment (PaulRitter) [inline Content.Server/GameTicking/GameTicker.Lobby.cs:28]: wait what

File: Content.Server/GameTicking/Rules/GameRuleSystem.cs [hunk 3]
```diff
@@ -20,7 +20,7 @@ public abstract class GameRuleSystem : EntitySystem
     ///     When the GameRule prototype with this ID is added, this system will be enabled.
     ///     When it gets removed, this system will be disabled.
     /// </summary>
-    public abstract string Prototype { get; }
+    public new abstract string Prototype { get; }
```
Comment (PaulRitter) [inline Content.Server/GameTicking/Rules/GameRuleSystem.cs:23]: wait what

File: Content.Shared/Throwing/ThrownItemSystem.cs [hunk 4]
```diff
@@ -54,17 +55,19 @@ private void OnHandleState(EntityUid uid, ThrownItemComponent component, ref Com
...
                 Logger.Error($"Found existing throwing fixture on {component.Owner}");
                 return;
             }
-
-            var shape = physicsComponent.Fixtures[0].Shape;
-            _fixtures.CreateFixture(physicsComponent, new Fixture(physicsComponent, shape) {CollisionLayer = (int) CollisionGroup.ThrownItem, Hard = false, ID = ThrowingFixture});
+            var fixture = fixturesComponent.Fixtures.Values.First();
+            var shape = fixture.Shape;
+            var throwingFixture = new Fixture(physicsComponent, shape) { CollisionLayer = (int) CollisionGroup.ThrownItem, Hard = false, ID = ThrowingFixture };
+            _fixtures.CreateFixture(physicsComponent, throwingFixture, true, fixturesComponent);
```
Comment (PaulRitter) [inline Content.Shared/Throwing/ThrownItemSystem.cs:70]: why set update to true? just use a named parameter here if you want to pass fixturesComponent, like so:
```suggestion
            _fixtures.CreateFixture(physicsComponent, throwingFixture, manager: fixturesComponent);
```

**Review by PaulRitter — CHANGES_REQUESTED** [review]

File: Content.Server/Chemistry/ReactionEffects/AreaReactionEffect.cs (hunk 1 above)
Comment (wrexbe) [reply Content.Server/Chemistry/ReactionEffects/AreaReactionEffect.cs:50]: I bugged Radrark#1226 on discord

File: Content.Server/GameTicking/GameTicker.Lobby.cs (hunk 2 above)
Comment (wrexbe) [reply Content.Server/GameTicking/GameTicker.Lobby.cs:28]: There is a Paused method on EntitySystem, so it is warning about it. By adding new, I am telling the compiler this is intentional.

File: Content.Server/GameTicking/Rules/GameRuleSystem.cs (hunk 3 above)
Comment (wrexbe) [reply Content.Server/GameTicking/Rules/GameRuleSystem.cs:23]: The warning was caused by "protected EntityPrototype? Prototype", so adding new tells the compiler that hiding the protected method was intentional.

File: Content.Server/Chemistry/ReactionEffects/AreaReactionEffect.cs (hunk 1 above)
Comment (wrexbe) [reply Content.Server/Chemistry/ReactionEffects/AreaReactionEffect.cs:50]: Mirrorcult says we don't need it

**Review by PaulRitter — APPROVED** [review]

**Comment by PaulRitter** [comment]
grafik

**Pull request merged**